### PR TITLE
Fix/landing styles

### PR DESCRIPTION
**Pull request by jesseng11**

### CODE REVIEW

File: src/app/main/main.controller.js [hunk 1]
```diff
@@ -8,15 +8,30 @@
...
         // if a 401 Unauthorized response is returned, then the token and user are 
         // cleared from local storage. This ensures that the user won't fall into an
         // infinite loop! If they are already authed, just redirect to discover
         if (TokenService.getToken() && AuthService.getUser()) {
           return $state.go('dashboard.discover');
         }
+        
+        vmMain.doStickToTop = false;
+        vmMain.winHeight = $window.innerHeight - 85; //minus height+padding of authentication navbar
+        vmMain.navTop = vmMain.winHeight + ".px";
```
Comment (drewbanin) [inline src/app/main/main.controller.js]: just curious -- why is this `.px` and not `px`?

**Comment by drewbanin** [comment]
one quick question but otherwise looks good to me!

File: src/app/main/main.controller.js (hunk 1 above)
Comment (jesseng11) [inline src/app/main/main.controller.js]: haha oops :P. fixed now!

Comment (jesseng11) [inline src/app/main/main.controller.js]: oh wait no, that broke it D:. ughhhhh whyyyyy

**Pull request merged**